src/Tickets/Libraries/Uplink/SLRController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * The Controller to set up the Uplink library.
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Libraries\Uplink
+ */
+
+namespace TEC\Tickets\Libraries\Uplink;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Common\StellarWP\Uplink\Register;
+use Tribe__Tickets__Main as Main;
+
+/**
+ * Controller for setting up the stellarwp/uplink library.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Libraries\Uplink
+ */
+class SLRController extends Controller_Contract {
+	/**
+	 * Plugin slug.
+	 *
+	 * @var string
+	 */
+	protected string $et_slr_plugin_slug = 'tec-seating';
+
+	/**
+	 * Plugin name.
+	 *
+	 * @var string
+	 */
+	protected string $et_slr_plugin_name = 'Seat Layouts & Reservations';
+
+	/**
+	 * Main plugin object.
+	 *
+	 * @var object
+	 */
+	protected object $et_main;
+
+	/**
+	 * Register the controller.
+	 *
+	 * @since TBD
+	 */
+	public function do_register(): void {
+		$this->et_main = tribe( 'tickets.main' );
+		$this->add_actions();
+	}
+
+	/**
+	 * Unregister the controller.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function unregister(): void {
+		$this->remove_actions();
+	}
+
+	/**
+	 * Add the action hooks.
+	 *
+	 * @since TBD
+	 */
+	public function add_actions(): void {
+		add_action( 'init', [ $this, 'register_plugin' ] );
+	}
+
+	/**
+	 * Remove the action hooks.
+	 *
+	 * @since TBD
+	 */
+	public function remove_actions(): void {
+		remove_action( 'init', [ $this, 'register_plugin' ] );
+	}
+
+	/**
+	 * Register the plugin in the uplink library.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function register_plugin(): void {
```

Comment:
If the SLR feature is deactivated using the constant set in the `wp-config.php` file, this registration will not happen since the `Uplink\SLR_Controller` is registered in the main SLR Controller.

Why not have this controller live in the `Seating` directory?
I would call the controller `Uplink`, register as it is registered now in the main Seating controller and be part of the SLR feature completely.